docs/build-tools/overview.md

```diff
@@ -0,0 +1,47 @@
+---
+id: overview
+title: Build Tools Overview
+sidebar_label: Overview
+---
+
+Metals works with the following text editors with varying degree of
```

Comment:
`text editors` you mean `build tools`, no ?

Reply:
Thanks! Fixed